lib/apple/index.js

```diff
@@ -89,6 +89,10 @@ exports.verifyPayment = function (payment, cb) {
 		});
 	}
 
+	if (typeof payment.secret !== 'undefined') {
```

Comment:
No biggy, but any reason for not just writing `if (payment.secret !== undefined) {`?